platforms/linux-armv6/Dockerfile

```
@@ -36,6 +36,7 @@ RUN \
     --no-modify-path \
     --profile minimal \
     && \
+  rustup default nightly && \
   rustup target add arm-unknown-linux-gnueabihf && \
```

```suggestion
    --profile minimal \
    --target arm-unknown-linux-gnueabihf \
    --default-toolchain nightly \
    && \
```

(I also changed it to use `--target ...`  instead of `rustup target add ...`)